src/components/Piechart.js

```diff
@@ -0,0 +1,75 @@
+import React, { PureComponent } from "react";
+import {
+  PieChart,
+  Pie,
+  Sector,
+  Cell,
+  ResponsiveContainer,
+  Tooltip
+} from "recharts";
+
+const data = [
+  { name: "Food", value: 5000 },
+  { name: "Cloth", value: 2000 },
+  { name: "Medical", value: 1050 }
+];
+
+const COLORS = ["#0088FE", "#00C49F", "#FFBB28"];
```

Comment:
You should use the same colours in both charts. I personally prefer these ones. You could probably use a shared colour palette imported from a config/settings outside your `components` folder? But it's not a critical issue as of now.